### ISSUE TITLE
Depencency tree in 4.0.0-alpha.0 cannot be resolved

### ISSUE THREAD

**Issue by mpetrunic** [ethereum/web3.js#5382]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

```
mpetrunic@mpetrunic-XPS-9320:~/Projects/ChainSafe/playground/web3.js$ yarn add web3@4.0.0-alpha.0
yarn add v1.22.19
[1/4] Resolving packages...
warning web3 > web3-eth@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth-personal > web3-eth@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth-contract > web3-eth@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth-ens > web3-eth@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth > web3-core-requestmanager@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-core > web3-providers-http@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth > web3-core-requestmanager > web3-providers-http@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-core > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth-abi > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth-iban > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth-personal > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth-contract > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth-ens > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-eth-accounts > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-net > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-core > web3-providers-ipc > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
warning web3 > web3-core > web3-providers-ws > web3-utils@4.0.0: Intended to be released as alpha, does not work in current form
Couldn't find any versions for "web3-providers-base" that matches "^1.0.0"
? Please choose a version of "web3-providers-base" from this list: (Use arrow keys)
Couldn't find any versions for "web3-providers-base" that matches "^1.0.0"
? Please choose a version of "web3-providers-base" from this list: (Use arrow keys)
❯ 1.0.0-alpha.1 
```

### Expected Behavior

Install without warnings or errors

### Steps to Reproduce

yarn add web3@4.0.0-alpha.0

### Web3.js Version

4.0.0-alpha.0

### Environment

- Operating System: Ubuntu
- Browser: -
- Node.js Version: 16
- NPM Version: yarn1

**Comment by mpetrunic** [comment]
Seems like 4.0.0 version of some packages was published a year ago and then deprecated: https://www.npmjs.com/package/web3-eth/v/4.0.0

Short term fix:
- remove version range to force resolution of alpha package

Long term:
open support ticket with npm to remove those 4.0.0 versions as they will prevent us from publishing 4.0.0 cc @GregTheGreek

**Comment by GregTheGreek** [comment]
I've emailed NPM to remove the following package and respective versions:
```
web3-eth@4.0.0
web3-utils@4.0.0
web3-core-requestmanager@4.0.0
web3-providers-http@4.0.0
```

**Comment by jdevcs** [comment]
Fix for 4.0.0-alpha.0 :

 Adding following explicitly to package.json users can fix yarn to skip using old deprecated packages and use specified ones:
```
"resolutions": {
    "web3-providers-http": "4.0.0-alpha.1",
    "web3-utils": "4.0.0-alpha.1",
    "web3-eth": "4.0.0-alpha.1"
  }
```
This resolves dependency tree with: `yarn add web3@4.0.0-alpha.0 `

We will also post alpha patch bump, so users will not need above mentioned temp fix with that.

**Comment by jdevcs** [comment]
4.x Alpha patch bump has been published:
https://www.npmjs.com/package/web3/v/4.0.1-alpha.0
https://github.com/ChainSafe/web3.js/releases/tag/v4.0.1-alpha.0
With this yarn is not resolving to deprecated package.  `yarn add web3@4.0.1-alpha.0`
